sql/core/benchmarks/DateTimeRebaseBenchmark-jdk21-results.txt

```diff
@@ -6,109 +6,109 @@ OpenJDK 64-Bit Server VM 21.0.4+7-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 Save DATE to parquet:                     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-after 1582, noop                                  11193          11193           0          8.9         111.9       1.0X
-before 1582, noop                                  7070           7070           0         14.1          70.7       1.6X
-after 1582, rebase EXCEPTION                      19836          19836           0          5.0         198.4       0.6X
-after 1582, rebase LEGACY                         19368          19368           0          5.2         193.7       0.6X
-after 1582, rebase CORRECTED                      19627          19627           0          5.1         196.3       0.6X
-before 1582, rebase LEGACY                        16301          16301           0          6.1         163.0       0.7X
-before 1582, rebase CORRECTED                     15612          15612           0          6.4         156.1       0.7X
+after 1582, noop                                  11734          11734           0          8.5         117.3       1.0X
+before 1582, noop                                  7155           7155           0         14.0          71.6       1.6X
+after 1582, rebase EXCEPTION                      20374          20374           0          4.9         203.7       0.6X
+after 1582, rebase LEGACY                         19681          19681           0          5.1         196.8       0.6X
+after 1582, rebase CORRECTED                      20160          20160           0          5.0         201.6       0.6X
+before 1582, rebase LEGACY                        16245          16245           0          6.2         162.5       0.7X
+before 1582, rebase CORRECTED                     16039          16039           0          6.2         160.4       0.7X
```

Comment:
https://github.com/apache/spark/blob/6099de7f5995b438335a5a545d5cf62dd3d68f0d/sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DateTimeRebaseBenchmark.scala#L166

@MaxGekk 

It seems that the numIters for some bench cases is fixed at 1, so the Stdev is always 0 because there is only test result.